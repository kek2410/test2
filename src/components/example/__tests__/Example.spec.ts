import { shallowMount, createLocalVue } from "@vue/test-utils";
import Example from "../Example.vue";
import Vuetify from "vuetify";

const localVue = createLocalVue();

describe("Example.vue", (): void => {
  let vuetify: any;
  let wr: any;

  beforeEach((): void => {
    vuetify = new Vuetify();
    wr = shallowMount(Example, {
      localVue,
      vuetify
    });
  });

  it("Example.vue > toMatchSnapshot", (): void => {
    expect(wr.html()).toMatchSnapshot();
  });

  it("Example.vue > test1", (): void => {
    const h1 = wr.find("h1");

    expect(h1.text()).toBe("Hello World!");
  });
});
