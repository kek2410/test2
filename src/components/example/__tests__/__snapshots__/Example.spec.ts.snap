// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example.vue Example.vue > toMatchSnapshot 1`] = `<h1>Hello World!</h1>`;
